lib/Controller/ObjectsController.php

```diff
@@ -153,7 +165,14 @@ public function update(int $id): JSONResponse
         if (isset($data['id'])) {
             unset($data['id']);
         }
-        return new JSONResponse($this->objectEntityMapper->updateFromArray(id: (int) $id, object: $data));
+
+        // save it
+        $oldObject = $this->objectEntityMapper->find($id);
+        $objectEntity = $this->objectEntityMapper->updateFromArray(id: $id, object: $data);
+
+        $this->auditTrailMapper->createAuditTrail(new: $objectEntity, old: $oldObject);
```

Comment:
Just checking, but is objectEntity fully memory-separated from oldObject? Otherwise both new and old will end up being the same.